### DISCUSSION TITLE
Why my git account is blocked ?

### DISCUSSION THREAD

**Discussion by mybidaz** [vercel/community#4129] [Account Issue] [answered]
### Question

 I can't sing up.

**Comment by spike014** [comment]
Me too. Why? I didn't do anything.

**Reply by spike014** [reply]
By the way, I use continue with Github.

**Reply by rumonedotdev** [reply]
I also couldn't sign up has anyone figured this out yet

**Reply by hungchihsueh** [reply]
Me too!
@vercel-infra please help.

**Reply by starry-wang** [reply]
me too

**Reply by Chisom-Maduka** [reply]
I been having the same issue, did not violate any of their rules, I was just trying to sign up. @vercel can you please fix this

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't discuss account information in public because of security concerns. 

You may have received an email with more details about the error and instructions on next steps. Please check your spam or junk folder in case it was filtered out of your inbox. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by rumonedotdev** [comment]
I was able to access the account everything is okay now, thanks for
reaching out
